src/lmcontrol/segment.py

```diff
@@ -285,17 +331,18 @@ def crop_size(string):
             seg_images.append(segi)
             seg_masks.append(segm)
         except UnsegmentableError:
-            if args.save_unseg:
```

Comment:
Please keep keep save_unseg and make center_crop optional

Reply:
done